src/main/java/commands/diet/dietmanager/DietSessionSearch.java

```diff
@@ -1,48 +1,107 @@
 package commands.diet.dietmanager;
 
 import commands.Command;
+import diet.DateParser;
+import diet.dietmanager.DietManagerParser;
+import exceptions.InvalidDateFormatException;
+import exceptions.diet.InvalidSearchDateException;
+import exceptions.profile.InvalidCommandFormatException;
 import storage.diet.DietStorage;
 import ui.diet.dietmanager.DietManagerUi;
 
 import java.io.File;
+import java.time.LocalDateTime;
+import java.util.Date;
+import java.util.HashMap;
 import java.util.Objects;
 import java.util.logging.Level;
 
+import static seedu.duke.Constant.COMMAND_WORD_SEARCH;
+
 public class DietSessionSearch extends Command {
 
     static final String FILEPATH = "saves/diet/";
```

Comment:
The FILEPATH can consider using the path constants created in seedu.duke.Constant